yang/iproute2-tc-qdisc.yang

```diff
@@ -170,12 +264,13 @@ module iproute2-tc-qdisc {
                 description "Name of device 'interface' the qdisc is attached to.";
             }
             leaf parent {
+                ipr2cgen:oper-default-val "root";
                 type parent-id;
                 description "[parent_id 'e.g 8001:1' | root ] Location of the qdisc in the traffic control hierarchy. ";
             }
             leaf handle {
                 type string {
-                    pattern '\d+:';
+                    pattern '[0-9a-fA-F]+:';
```

Comment:
tried to check on this one, but could not find any documentation says that this is a hex number, are we sure about this ?